core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -646,126 +617,6 @@ public void testMultiValuedFieldOrderedByKeyDesc() throws Exception {
     }
 
     /**
-     * The script will change to document date values to the following:
-     * <p>
-     * doc 1: [ Feb 2, Mar 3]
-     * doc 2: [ Mar 2, Apr 3]
-     * doc 3: [ Mar 15, Apr 16]
-     * doc 4: [ Apr 2, May 3]
-     * doc 5: [ Apr 15, May 16]
-     * doc 6: [ Apr 23, May 24]
-     */
-    public void testMultiValuedFieldWithValueScript() throws Exception {
-        SearchResponse response = client().prepareSearch("idx")
-                .addAggregation(dateHistogram("histo")
-                        .field("dates")
-                        .script(new Script("new DateTime(_value, DateTimeZone.UTC).plusMonths(1).getMillis()"))
```

Comment:
I think we should keep those tests, since they are the only ones that test value scripts (when both a script and a field are provided). However we don't need to keep doing the date math, we could just use a script that returns _value directly?